src/ansys/geometry/core/modeler.py

```diff
@@ -304,7 +304,7 @@ def __repr__(self) -> str:
 
     @protect_grpc
     def run_discovery_script_file(
-        self, file_path: str, script_args: Dict[str, str], import_design=False
+        self, file_path: str, script_args: Dict[str, str] = {}, import_design=False
```

Comment:
Mmm.. not a fan of this. Default it to ``None``, add the ``Optional`` typehint and actively check in the code whether it is None or not. Do something like

```py
script_args = script_args if script_args else {}
```

Reason is that setting an empty dictionary as default value may lead to undesired behavior --> https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

I know we are not editing the dictionary inside, but it is just best to avoid these issues.